managedtenants/bundles/index_builder.py

```diff
@@ -35,11 +39,15 @@ def build_and_push(
         :return: An Index image that has been pushed.
         """
 
-        index_image = self._build(bundles, hash_string, skip_validation)
+        index_image = self._build(
+            bundles, hash_string, skip_validation, test_run
+        )
         return self._push(index_image)
 
     # pylint: disable=unused-argument
-    def _build(self, bundles, hash_string, skip_validation=False):
+    def _build(
+        self, bundles, hash_string, skip_validation=False, test_run=False
+    ):
```

Comment:
yep, pls group all refactorings in a separate commit, so reviews can go straight to what's important :)